experimental/Schemes/IdealSheaves.jl

```diff
@@ -572,7 +572,7 @@ end
 @doc raw"""
     maximal_associated_points(I::IdealSheaf)
 
-Returns a `Vector` of `IdealSheaf`s corresponding to the non-embedded associated points of ``I`` on ``scheme(I)``.
+Return a `Vector` of `IdealSheaf`s corresponding to the non-embedded associated points of ``I`` on ``scheme(I)``.
```

Comment:
So yeah maybe it should be `scheme(I)` with a single backtick, but I deliberately tried not to mix in such changes, too, because I'd be opening a bottom less pit 😂 . We'd first have to decide what the rules when to use what should be...